pupil-spa/src/app/warmup-intro/warmup-intro.component.html

```diff
@@ -0,0 +1,10 @@
+<header class="page-header">
+  <h1 class="heading-xlarge">Warm-up questions</h1>
+</header>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <p class="lede">Press 'Start now' to see 3 warm-up questions. Use them to familiarise yourself with what the questions will look like and how to enter your answers.</p>
+    <a class="button button-start" role="button" id="start-now-button" href="warm-up-start">Start now</a>
```

Comment:
@jon-shipley for consistency i replaced it with a button and a click handler for now